plugin/src/main/java/net/dzikoysk/funnyguilds/config/PluginConfiguration.java

```diff
@@ -381,10 +381,13 @@ public class PluginConfiguration extends OkaeriConfig {
     @Comment("Funkcja niedostępna, jeśli możliwość teleportacji do gildii jest wyłączona")
     public boolean escapeEnable = true;
 
-    @Min(0)
+    @PositiveOrZero
     @Comment("")
-    @Comment("Czas w sekundach, jaki musi upłynąć od włączenia ucieczki do teleportacji")
-    public int escapeDelay = 120;
+    @Comment("Czas jaki musi upłynąć od włączenia ucieczki do teleportacji")
+    @Comment("Format: <wartość><jednostka><wartość><jednostka><...>")
+    @Comment("Jednostki: ns - nanosekundy, ms - milisekundy, s - sekundy, m - minuty, h - godziny, d - dni")
```

Comment:
Point in ns/ms where??? 🐒

Reply:
What point? Do you mean `ns.` and `ms.`?

Reply:
As in `there is no point in providing ns and ms in the example`.